package.json

```diff
@@ -21,7 +21,7 @@
     "babel-preset-es2015": "6.9.0",
     "babel-preset-stage-2": "^6.5.0",
     "babel-register": "^6.9.0",
-    "chai": "^3.5.0",
+    "expect": "^1.20.2",
```

Comment:
Familiarity primarily, I'm used to work with expect and it's minimalistic and the creator is a really smart and prominent figure in the React community, but if you disagree I can change back :)